### PR TITLE
Adding documentation for setting custom modifier class name

### PR DESCRIPTION
**Pull request by stephenhmarsh**

### CODE REVIEW

**Comment by johnnyshields** [comment]
@dblock wait I thought we removed this feature in favor of Mongoid::Userstamp?

**Comment by stephenhmarsh** [comment]
@johnnyshields - FYI I am using Userstamps, but get ``uninitialized constant User`` trying to track modifier when making changes to nested documents scoped to a parent.  Partial stacktrace:

```
14:52:33 web.1       | NameError - uninitialized constant User:
14:52:33 web.1       |   activesupport (4.1.8) lib/active_support/dependencies.rb:519:in `load_missing_constant'
14:52:33 web.1       |   activesupport (4.1.8) lib/active_support/dependencies.rb:180:in `const_missing'
14:52:33 web.1       |   activesupport (4.1.8) lib/active_support/inflector/methods.rb:238:in `block in constantize'
14:52:33 web.1       |   activesupport (4.1.8) lib/active_support/inflector/methods.rb:236:in `constantize'
14:52:33 web.1       |   activesupport (4.1.8) lib/active_support/core_ext/string/inflections.rb:66:in `constantize'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-60c4d942df30/lib/mongoid/relations/metadata.rb:575:in `klass'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-60c4d942df30/lib/mongoid/relations/builders/referenced/in.rb:19:in `build'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-60c4d942df30/lib/mongoid/relations/accessors.rb:43:in `create_relation'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-60c4d942df30/lib/mongoid/relations/accessors.rb:26:in `__build__'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-60c4d942df30/lib/mongoid/relations/accessors.rb:104:in `block (2 levels) in get_relation'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-60c4d942df30/lib/mongoid/threaded/lifecycle.rb:125:in `_loading'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-60c4d942df30/lib/mongoid/relations/accessors.rb:100:in `block in get_relation'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-60c4d942df30/lib/mongoid/threaded/lifecycle.rb:84:in `_building'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-60c4d942df30/lib/mongoid/relations/accessors.rb:99:in `get_relation'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-60c4d942df30/lib/mongoid/relations/accessors.rb:187:in `block in getter'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-history-015838cef1dc/lib/mongoid/history/trackable.rb:217:in `history_tracker_attributes'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-history-015838cef1dc/lib/mongoid/history/trackable.rb:268:in `track_history_for_action'
14:52:33 web.1       |    () Users/stephenmarsh/.rvm/gems/ruby-2.2.0/bundler/gems/mongoid-history-015838cef1dc/lib/mongoid/history/trackable.rb:232:in `track_update'
```

**Comment by johnnyshields** [comment]
OK. I'm swamped at work right now but once things even out next week I'll take a look at this. (I'm a maintainer of Mongoid::Userstamp as well). Thanks for PR.

**Comment by stephenhmarsh** [comment]
Welcome. Thanks. Will let you know if I come up with any more info on this.

File: README.md [hunk 1]
```diff
@@ -238,6 +238,27 @@ instructions above then run the following command:
...
+# config/initializers/mongoid-history.rb
+# initializer for mongoid-history
+
+Mongoid::History.modifier_class_name = 'Author'
+```
+
+Or perhaps you are namespacing to a module:
+
+```ruby
+Mongoid::History.modifier_class_name = 'CMS::User'
```
Comment (dblock) [inline README.md]: This should be `CMS::Author`, right?

**Comment by dblock** [comment]
Can you please squash your commits after fixing my note above? Thanks!

**Comment by stephenhmarsh** [comment]
@dblock done and done.

**Comment by dblock** [comment]
I like it, merging.

**Pull request merged**